test/test_codegen.py

```diff
@@ -174,37 +188,65 @@ def test_get_temperature(mechname):
         assert np.abs(t - t_pm) < tol
 
 
-@pytest.mark.parametrize("mechname", ["uiuc", "sanDiego"])
-def test_get_thermo_properties(mechname):
+@pytest.mark.parametrize("mechname,fuel", [("uiuc", "C2H4"),
+                                       ("sanDiego", "H2")])
+def test_kinetics(mechname, fuel):
     """This function tests that pyrometheus-generated code
-    computes thermodynamic properties c_p, s_R, h_RT, and k_eq
-    correctly by comparing against Cantera"""
-    # Create Cantera and pyrometheus objects
+    computes the right rates of progress for given temperature
+    and composition"""
```

Comment:
Specify "right". (I know you didn't add this, but now is as good a time as any.)

Reply:
I don't understand what you are asking for.

Reply:
I took this to mean you wanted to clarify that "right" means.   In this case right = Cantera-predicted.

Reply:
4b6c51b